src/index.js

```diff
@@ -489,14 +491,15 @@ export default class Proctor {
   }
 
   sendEvents() {
-    if (!this.eventsConfig.url) return;
+    if (!this.baseUrl || !this.eventsConfig.endpoint) return;
     if (this.recordedViolationEvents.length === 0) return;
 
+    const url = `${this.baseUrl}${this.eventsConfig.endpoint}`;
```

Comment:
Don't directly append it like this. There are methods available to combine URLs. use that